python/rascaline/tests/utils/splines.py

```diff
@@ -102,3 +111,261 @@ def test_splines_numerical_derivative_error():
     match = "Numerically derivative of the radial integral can not be performed"
     with pytest.raises(ValueError, match=match):
         RadialIntegralFromFunction(**kwargs).compute()
+
+
+def test_kspace_radial_integral():
+    """Test against anayliycal integral with Gaussian densities and GTOs"""
+
+    cutoff = 2
+    max_radial = 6
+    max_angular = 3
+    atomic_gaussian_width = 1.0
+    k_cutoff = 1.2 * np.pi / atomic_gaussian_width
+
+    basis = GtoBasis(cutoff=cutoff, max_radial=max_radial)
+
+    spliner = KSpaceSpliner(
+        max_radial=max_radial,
+        max_angular=max_angular,
+        k_cutoff=k_cutoff,
+        basis=basis,
+        density=DeltaDensity(),  # density does not enter in a Kspace radial integral
```

Comment:
if the atomic density does not enter the K-space RI, should we remove the parameter from the constructor?

Reply:
But is does for the center contribution which we are not testing here.